CONTRIBUTING.md

```diff
@@ -128,6 +128,20 @@ make server
 Any time changes are made to the Go code, you will have to stop the server and run the above command again.
 Once the Meshery server is up and running, you should be able to access Meshery on your `localhost` on port `9081` at `http://localhost:9081`.
 
+**Please note**: If you get error while starting the server as **"Meshery Development Incompatible"** then follow the below guideline 👇
+
+<a href="https://user-images.githubusercontent.com/86982322/246590595-22177454-4b04-4c01-aab6-546ff9d4a318.png" ><img src="https://user-images.githubusercontent.com/86982322/246590595-22177454-4b04-4c01-aab6-546ff9d4a318.png" width="50%"></a>
```

Comment:
This might be your user profile where this image is uploaded and linked, please add the image here and reference it from the same: https://github.com/meshery/meshery/tree/master/.github/assets/images

Reply:
Sure, I have updated
Can you please check?